components/integrations/abstract_outgoing_webhook.jsx

```diff
@@ -310,7 +310,7 @@ export default class AbstractOutgoingWebhook extends React.Component {
                                 <div className='form__help'>
                                     <FormattedMessage
                                         id='add_outgoing_webhook.description.help'
-                                        defaultMessage='Description for your incoming webhook.'
+                                        defaultMessage='Describe your outgoing webhook.'
```

Comment:
outgoing vs incoming?

Reply:
the message ID seems to be for an outgoing webhook, and was in the outgoing webhooks section, so I changed it to 'Describe your outgoing webhook'. Should this be 'incoming'?

Reply:
If the text is in fact for outgoing webhooks then definitely, thanks for finding that bug!